### ISSUE TITLE
A11y_SQL Azure Data Studio_Extensions - install_Screen reader: After invoking "Install" button Screen Reader is not announcing "Installation" status.

### ISSUE THREAD

**Issue by PadminiGariki** [dotnet/maui#16817]
### Description

[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG4.1.3; #ScreenReader; #Win32; #Benchmark; #FTP; #STP;

### Environment Details:​
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11

### Steps to Reproduce

1. Launch Azure Data Studio.
2. From the left menu, click the 'Extensions' icon to open the Extension Manager.
3. Choose any extension from the library.
4. Click the green 'Install' button to install. 
5. Now verify that NVDA/Narrator is not announcing "Installation" status.

### Actual Results:
After invoking "Installation" button Screen Reader is not announcing "Installation" status whether it is installed or not.
**Note:**
**1.Similar issue observed with chrome+ NVDA.**
**2.Similar issue is also reproducing in the below scenarios:**
**1. Issue: Extensions - Uninstall**
1. Launch Azure Data Studio.
2. From the left menu, click the 'Extensions' icon to open the Extension Manager.
3. Choose any extension from the library.
4. Click the green 'Install' button to install. Blue 'Uninstall' button should now appear.
5. Click the blue 'Uninstall' button to uninstall.
7. Now verify that NVDA/Narrator is not announcing "Uninstall" status.

**2.Issue: New Terminal**
1. Launch Azure Data Studio.
2. Navigate to the menu and click on View > Terminal
3. Click on the "Plus" sign
4. Now verify that NVDA/Narrator is not announcing whether it is created or not.

### Expected Results:
After invoking "Installation" button Screen Reader should announce "Installation" status whether it is installed or not.

### User Impact:
Screen reader user will not be able to know installation status whether it is installed or not. If NVDA/Narrator not announcing.

### Attachment:

[Install and Uninstall NVDA .webm](https://github.com/dotnet/maui/assets/142479541/cc623e4b-7bab-4922-af2e-288d2a79c201)

[install and uninstall.webm](https://github.com/dotnet/maui/assets/142479541/15f4640e-3fa9-452a-b0c7-d0a06f4485bc)

<img width="960" alt="MicrosoftTeams-image (3)" src="https://github.com/dotnet/maui/assets/142479541/9c8726b9-2eaa-4199-ae06-6f2cf8f7d3a3">

[NVDA.webm](https://github.com/dotnet/maui/assets/142479541/f5bb78a9-8ad7-4daf-9094-523e1366c274)

[Narrator.webm](https://github.com/dotnet/maui/assets/142479541/2e0f73af-a3d5-4799-805e-c31a5dea00bf)

<img width="960" alt="MicrosoftTeams-image (1)" src="https://github.com/dotnet/maui/assets/142479541/edc20bb8-3a4a-4185-af9e-dceba9f48958">

### Version with bug

8.0.92

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

**Comment by PureWeen** [comment]
Wrong Repository

**Comment by PadminiGariki** [comment]
Sorry Inconvenience caused. Adding #Closed; tag. Will file issue on Correct Repo.
